Rust integration tests fails to build.

Rust Integration tests fail to build because of workspaces 0.4.1 dependency.

```
$ npm run test:integration

....


error[E0308]: mismatched types
  --> /home/$USER/.cargo/registry/src/github.com-1ecc6299db9ec823/workspaces-0.4.1/src/rpc/client.rs:60:9
   |
60 | /         retry(|| async {
61 | |             let result = self.rpc_client.call(method).await;
62 | |             match &result {
63 | |                 Ok(response) => {
...  |
93 | |         })
94 | |         .await
   | |______________^ expected struct `FinalExecutionOutcomeView`, found struct `near_primitives::views::FinalExecutionOutcomeView`
   |
   = note: expected enum `Result<FinalExecutionOutcomeView, JsonRpcError<near_jsonrpc_primitives::types::transactions::RpcTransactionError>>`
              found enum `Result<near_primitives::views::FinalExecutionOutcomeView, JsonRpcError<near_jsonrpc_client::methods::broadcast_tx_commit::RpcTransactionError>>`
   = note: perhaps two different versions of crate `near_primitives` are being used?

...

Some errors have detailed explanations: E0277, E0308.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `workspaces` due to 32 previous errors
```

Steps to reproduce:
```
$ npx create-near-app
======================================================
👋 Welcome to NEAR! Learn more: https://docs.near.org/
🔧 Let's get your dApp ready.
======================================================
(NEAR collects anonymous information on the commands used. No personal information that could identify you is shared)

✔ Select your smart-contract language › Rust
✔ Select language for Sandbox Test › Rust Sandbox Tests
✔ Select a template for your frontend › No frontend
✔ Name your project (this will create a directory with that name) … test-failure
✔ Run 'npm install' now? … yes
```

--- a/templates/integration-tests/rust-tests/Cargo.toml
+++ b/templates/integration-tests/rust-tests/Cargo.toml
@@ -14,7 +14,7 @@ serde_json = { version = "1.0", features = ["arbitrary_precision"] }
 tokio = { version = "1.18.1", features = ["full"] }
 tracing = "0.1"
 tracing-subscriber = { version = "0.3.11", features = ["env-filter"] }
-workspaces = "0.4.0"
+workspaces = "0.6.0"
 pkg-config = "0.3.1"
 
 [[example]]

--- a/templates/integration-tests/rust-tests/src/tests.rs
+++ b/templates/integration-tests/rust-tests/src/tests.rs
@@ -1,8 +1,7 @@
 use std::{env, fs};
 use near_units::parse_near;
 use serde_json::json;
-use workspaces::prelude::*;
-use workspaces::{network::Sandbox, Account, Contract, Worker};
+use workspaces::{Account, Contract};
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
@@ -16,26 +15,25 @@ async fn main() -> anyhow::Result<()> {
     // create accounts
     let account = worker.dev_create_account().await?;
     let alice = account
-        .create_subaccount(&worker, "alice")
+        .create_subaccount( "alice")
         .initial_balance(parse_near!("30 N"))
         .transact()
         .await?
         .into_result()?;
 
     // begin tests
-    test_default_message(&alice, &contract, &worker).await?;
-    test_changes_message(&alice, &contract, &worker).await?;
+    test_default_message(&alice, &contract).await?;
+    test_changes_message(&alice, &contract).await?;
     Ok(())
 }
 
 async fn test_default_message(
     user: &Account,
     contract: &Contract,
-    worker: &Worker<Sandbox>,
 ) -> anyhow::Result<()> {
     let message: String = user
-        .call(&worker, contract.id(), "get_greeting")
-        .args_json(json!({}))?
+        .call( contract.id(), "get_greeting")
+        .args_json(json!({}))
         .transact()
         .await?
         .json()?;
@@ -48,16 +46,16 @@ async fn test_default_message(
 async fn test_changes_message(
     user: &Account,
     contract: &Contract,
-    worker: &Worker<Sandbox>,
 ) -> anyhow::Result<()> {
-    user.call(&worker, contract.id(), "set_greeting")
-        .args_json(json!({"message": "Howdy"}))?
+    user.call(contract.id(), "set_greeting")
+        .args_json(json!({"message": "Howdy"}))
         .transact()
-        .await?;
+        .await?
+        .into_result()?;
 
     let message: String = user
-        .call(&worker, contract.id(), "get_greeting")
-        .args_json(json!({}))?
+        .call(contract.id(), "get_greeting")
+        .args_json(json!({}))
         .transact()
         .await?
         .json()?;

